KinEcosystem/BI/KBIEvent.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  KBIEvent.swift
+//  KinEcosystem
+//
+//  Created by Elazar Yifrach on 28/06/2018.
+//  Copyright © 2018 Kik Interactive. All rights reserved.
+//
+
+import Foundation
+
+protocol KBIEvent: Codable {
+    var common: Common { get }
```

Comment:
why only common? do we need user and client as well?

Reply:
This is only serving a generic function. I exposed only what was needed (eventId from common)